[O11y][Apache Spark] Remove unnecessary filter from the visualizations

harnish-elastic opened the pull request:
- Bug
<!-- Type of change
Please label this PR with one of the following labels, depending on the scope of your change:
- Bug
- Enhancement
- Breaking change
- Deprecation
-->

## What does this PR do?

- Remove unnecessary filters from the visualizations
- Update aggregation from maximum to last_value

#### Note

- Since the look of dashboard is not changed, no need to update dashboard screenshot

<!-- Mandatory
Explain here the changes you made on the PR.
-->

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have added an entry to my package's `changelog.yml` file.

<!--
## Author's Checklist

 Recommended
Add a checklist of things that are required to be reviewed in order to have the PR approved
-->

<!-- 
## How to test this PR locally

Recommended
Explain here how this PR will be tested by the reviewer: commands, dependencies, steps, etc.
-->

## Related issues

<!-- Recommended
Link related issues below. Insert the issue link or reference after the word "Closes" if merging this should automatically close it.

- Closes #123
- Relates #123
- Requires #123
- Supersedes #123
-->
- Closes #7409
- Relates #7406 

<!-- 
## Screenshots

Optional
Add here screenshots presenting:
- Kibana UI forms presenting configuration options exposed by the integration
- dashboards with collected metrics or logs
-->

elasticmachine commented:
## :green_heart: Build Succeeded


<!-- BUILD BADGES-->
> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-7467/detail/PR-7467/1//pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-7467/detail/PR-7467/1//tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-7467/detail/PR-7467/1//changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-7467/detail/PR-7467/1//artifacts) [![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](http://integrations_7467.docs-preview.app.elstc.co/diff)  [![preview](https://img.shields.io/badge/elastic-observability-blue)](https://ci-stats.elastic.co/app/apm/services/fleet-ci/transactions/view?rangeFrom=2023-08-21T06:15:41.798Z&rangeTo=2023-08-21T06:35:41.798Z&transactionName=BUILD+Ingest-manager%2Fintegrations%2FPR-%7Bnumber%7D&transactionType=job&latencyAggregationType=avg&traceId=020978d823181b3a1fd079216f8dc4cb&transactionId=4c4764f8390e7eae)

<!-- BUILD SUMMARY-->
<details><summary>Expand to view the summary</summary>
<p>

#### Build stats




* Start Time: 2023-08-21T06:25:41.798+0000


* Duration: 19 min 44 sec




#### Test stats :test_tube:

| Test         | Results                         |
| ------------ | :-----------------------------: |
| Failed       | 0  |
| Passed       | 15  |
| Skipped      | 0 |
| Total        | 15   |


</p>
</details>

<!-- TEST RESULTS IF ANY-->


<!-- STEPS ERRORS IF ANY -->














## :robot: GitHub comments

<details><summary>Expand to view the GitHub comments</summary>
<p>

To re-run your PR in the CI, just comment with:



- `/test` : Re-trigger the build.




</p>
</details>

<!--COMMENT_GENERATED_WITH_ID_comment.id-->

elasticmachine commented:
### :globe_with_meridians: Coverage report

Name | Metrics % (`covered/total`) | Diff
 --- | --- | ---
Packages | 100.0% (`0/0`) | :green_heart: 
Files | 100.0% (`0/0`) | :green_heart: 3.367
Classes | 100.0% (`0/0`) | :green_heart: 3.367
Methods | 75.0% (`12/16`) | :-1: -16.827
Lines | 100.0% (`0/0`) | :green_heart: 11.669
Conditionals | 100.0% (`0/0`) | :green_heart: 

<!--COMMENT_GENERATED_WITH_ID_coverage-->

rajvi-elastic reviewed (approved):
Looks good to me!

muthu-mps commented on packages/apache_spark/changelog.yml:
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.6.1"
+  changes:
+    - description: Remove unnecessary filter from the visualizations
```
```suggestion
    - description: Remove incorrect filter from the visualizations
```

muthu-mps commented:
@harnish-elastic -  Can you please add the context on why we changed the aggregation from maximum to last value?

harnish-elastic commented:
> @harnish-elastic - Can you please add the context on why we changed the aggregation from maximum to last value?

Updated the description [here](https://github.com/elastic/integrations/pull/7467#:~:text=from%20the%20visualizations-,The%20fields,-that%20were%20used). Thank you!

muthu-mps reviewed (approved):
Looks good!

elasticmachine commented:
Package apache_spark - 0.6.1 containing this change is available at https://epr.elastic.co/search?package=apache_spark
